### PR TITLE
Update schedule-replicas-by-topology-labels.md

### PR DESCRIPTION
**Pull request by qiancai**
1. Add tidb server label example
2. Move the recommended TiUP method before the CLI/Configuration file methods

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add tidb server labels example

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

https://docs.pingcap.com/zh/tidb/stable/tidb-configuration-file#labels

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/17208
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: schedule-replicas-by-topology-labels.md [hunk 1]
```diff
@@ -14,9 +14,156 @@ To improve the high availability and disaster recovery capability of TiDB cluste
 
 To make this mechanism effective, you need to properly configure TiKV and PD so that the topology information of the cluster, especially the TiKV location information, is reported to PD during deployment. Before you begin, see [Deploy TiDB Using TiUP](/production-deployment-using-tiup.md) first.
 
-## Configure `labels` based on the cluster topology
+## Configure `labels` for TiKV, TiFlash, and TiDB
 
-### Configure `labels` for TiKV and TiFlash
+You can configure `labels` for TiKV, TiFlash, and TiDB based on the cluster topology.
+
+### Configure a cluster using TiUP (recommended)
```
Comment (qiancai) [inline schedule-replicas-by-topology-labels.md:21]: Tip for reviewers: In the original content, this section is in L130. This PR adds info about TiDB node configuration to this section, and moves this section to the beginning of this doc as this method is recommended.

File: schedule-replicas-by-topology-labels.md [hunk 2]
```diff
@@ -14,9 +14,156 @@ To improve the high availability and disaster recovery capability of TiDB cluste
...
+
+When using TiUP to deploy a cluster, you can configure the TiKV location in the [initialization configuration file](/production-deployment-using-tiup.md#step-3-initialize-cluster-topology-file). TiUP will generate the corresponding configuration files for TiDB, TiKV, PD, and TiFlash during deployment.
+
+In the following example, a two-layer topology of `zone/host` is defined. The TiDB node, TiKV nodes, and TiFlash nodes of the cluster are distributed among three zones, z1, z2, and z3.
```
Comment (qiancai) [inline schedule-replicas-by-topology-labels.md:25]:
```suggestion
In the following example, a two-layer topology of `zone/host` is defined. The TiDB nodes, TiKV nodes, and TiFlash nodes of the cluster are distributed among three zones, z1, z2, and z3.
```

File: schedule-replicas-by-topology-labels.md [hunk 3]
```diff
@@ -14,9 +14,156 @@ To improve the high availability and disaster recovery capability of TiDB cluste
...
 
-## Configure `labels` based on the cluster topology
+## Configure `labels` for TiKV, TiFlash, and TiDB
 
-### Configure `labels` for TiKV and TiFlash
+You can configure `labels` for TiKV, TiFlash, and TiDB based on the cluster topology.
+
+### Configure a cluster using TiUP (recommended)
+
+When using TiUP to deploy a cluster, you can configure the TiKV location in the [initialization configuration file](/production-deployment-using-tiup.md#step-3-initialize-cluster-topology-file). TiUP will generate the corresponding configuration files for TiDB, TiKV, PD, and TiFlash during deployment.
```
Comment (hfxsd) [inline schedule-replicas-by-topology-labels.md:23]: 这里的 PD 要删掉吗？

**Review by hfxsd — APPROVED** [review]

File: schedule-replicas-by-topology-labels.md (hunk 3 above)
Comment (qiancai) [reply schedule-replicas-by-topology-labels.md:23]: @pepezzzz, PTAL

Comment (pepezzzz) [reply schedule-replicas-by-topology-labels.md:23]: 建议保留，因为要结合    pd: replication.location-labels: 才能正确工作。

**Comment by pepezzzz** [comment]
LGTM

**Pull request merged**